### PR TITLE
Package adelfa.0.1

### PR DESCRIPTION
**Pull request by cjohnson19**
### `adelfa.0.1`
Proof assistant for reasoning about LF specifications

---
* Homepage: http://sparrow.cs.umn.edu/adelfa/index.html
* Source repo: git+https://github.com/adelfa-prover/adelfa.git
* Bug tracker: https://github.com/adelfa-prover/adelfa/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

File: packages/adelfa/adelfa.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+opam-version: "2.0"
+synopsis: "Proof assistant for reasoning about LF specifications"
+maintainer: ["Chase Johnson <joh13266@umn.edu>"]
+authors: ["Mary Southern" "Chase Johnson <joh13266@umn.edu>"]
+license: "GPL-3.0"
+homepage: "http://sparrow.cs.umn.edu/adelfa/index.html"
+doc: "http://sparrow.cs.umn.edu/adelfa/reference-guide.html"
+bug-reports: "https://github.com/adelfa-prover/adelfa/issues"
+depends: [
+  "dune" {>= "3.0"}
```
Comment (mseri) [inline packages/adelfa/adelfa.0.1/opam:11]:
```suggestion
  "ocaml" {>= "4.12"}
  "dune" {>= "3.0"}
```
Due to
```
#=== ERROR while compiling adelfa.0.1 =========================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.10.2 | pinned(https://github.com/adelfa-prover/adelfa/archive/refs/tags/0.1.tar.gz)
# path                 ~/.opam/4.10/.opam-switch/build/adelfa.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p adelfa -j 39 @install
# exit-code            1
# env-file             ~/.opam/log/adelfa-7-800f24.env
# output-file          ~/.opam/log/adelfa-7-800f24.out
### output ###
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.core.objs/byte -no-alias-deps -o src/.core.objs/byte/extensions.cmo -c -impl src/extensions.ml)
# File "src/extensions.ml", line 392, characters 6-14:
# 392 |       Seq.cons
#             ^^^^^^^^
# Error: Unbound value Seq.cons
# (cd _build/default && /home/opam/.opam/4.10/bin/ocamlyacc src/parser.mly)
# 2 shift/reduce conflicts.
```
and a few other features of the new stldib and compiler

**Comment by mseri** [comment]
No need to make any change to the PR or the tagged sources for this fix, but please update the HEAD of your master branch with these changes

File: packages/adelfa/adelfa.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Proof assistant for reasoning about LF specifications"
+maintainer: ["Chase Johnson <joh13266@umn.edu>"]
+authors: ["Mary Southern" "Chase Johnson <joh13266@umn.edu>"]
+license: "GPL-3.0"
```
Comment (mseri) [inline packages/adelfa/adelfa.0.1/opam:6]:
```suggestion
license: "GPL-3.0-only"
```

**Comment by cjohnson19** [comment]
Thanks for the fixes! I've updated the master with these changes. I also had an outdated homepage reference which I didn't catch before.

**Pull request merged**